### PR TITLE
Add past events

### PR DESCRIPTION
**Pull request by Brilator**

### CODE REVIEW

**Comment by Brilator** [comment]
Needs some more detail, but could be merged to reduce clutter.

File: src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    start: 2023-05-31T10:00:00
+    end: 2023-05-31T11:30:00
+  - 
+    start: 2023-06-14T10:00:00
+    end: 2023-06-14T11:30:00
+mode: On-site
+audience: [Users]
+location:
+  short: HHU, ZSL 22.07, Room U1.025
+  address: HHU, ZSL 22.07, Room U1.025
```
Comment (Freymaurer) [inline src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md:19]: Not used in frontmatter?

File: src/content/events/2023-08-17_ARC-Club_HHU.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Installation
+  - [ARCitect](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/ARCitect-Manual/index.html)
+  - [Swate](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/SwateManual/Docs01-Installing-Swate.html)
+  - [ARC Commander](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/ArcCommanderManual/index-setup.html)  
+- List of projects to be ARCified
+
+## Why you want to join the **ARC Club**
+
+- You've needed data before, now is your chance to make data available
+- Your contribution to the ARC will be visible
```
Comment (Freymaurer) [inline src/content/events/2023-08-17_ARC-Club_HHU.md:52]: I think we do not render github style emojies

File: src/content/events/2023-08-17_ARC-Club_HHU.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+- Drinks :beers:
+- After-ARCathon Party 🥳
+
+## Rough Schedule
+
+### Day 1
+
+Time | Content
+--- | ---
+Morning | - Tech-check (Installation) <br> - Intro ARC, ARC Commander & DataHUB <br> - hands-on with dummy data
```
Comment (Freymaurer) [inline src/content/events/2023-08-17_ARC-Club_HHU.md:63]: Avoid `<br />` as much as possible

File: src/content/events/2023-11-15_CEPLAS-ARC-Clubs.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  affiliation: CEPLAS Data
+  url: https://www.ceplas.eu/en/research/data-science-and-data-management
+image: ~/assets/images/events/ARCClub.drawio.png
+---
+
+
+## Goals
+
+- Move existing datasets into [ARCs](./../../../implementation/AnnotatedResearchContext.html)
+- Share them via the [DataHUB](./../../../implementation/DataHub.html)
```
Comment (Freymaurer) [inline src/content/events/2023-11-15_CEPLAS-ARC-Clubs.md:27]: Use urls based from root instead of relative

**Review by Freymaurer — CHANGES_REQUESTED** [review]

File: src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md (hunk 1 above)
Comment (Brilator) [reply src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md:19]: address you mean? 
I know, but we need it somewhere

File: src/content/events/2023-08-17_ARC-Club_HHU.md (hunk 2 above)
Comment (Brilator) [reply src/content/events/2023-08-17_ARC-Club_HHU.md:52]: Jup, all old copy / paste from knowledge base

File: src/content/events/2023-08-17_ARC-Club_HHU.md (hunk 3 above)
Comment (Brilator) [reply src/content/events/2023-08-17_ARC-Club_HHU.md:63]: I know.

File: src/content/events/2023-11-15_CEPLAS-ARC-Clubs.md (hunk 4 above)
Comment (Brilator) [reply src/content/events/2023-11-15_CEPLAS-ARC-Clubs.md]: https://github.com/nfdi4plants/nfdi4plants.github.io/pull/197/commits/d9dacdb3112ee0930f9861b024372dc0ef45073c

File: src/content/events/2023-08-17_ARC-Club_HHU.md (hunk 2 above)
Comment (Freymaurer) [reply src/content/events/2023-08-17_ARC-Club_HHU.md:52]: I could try adding support with this package: https://github.com/rhysd/remark-emoji

File: src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md (hunk 1 above)
Comment (Brilator) [reply src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md:19]: Agreed.

File: src/content/events/2023-08-17_ARC-Club_HHU.md (hunk 3 above)
Comment (Brilator) [reply src/content/events/2023-08-17_ARC-Club_HHU.md]: https://github.com/nfdi4plants/nfdi4plants.github.io/pull/197/commits/0bf7e22fbb6676dcd1ef021c6f0bb148d24ad40d

File: src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md (hunk 1 above)
Comment (Brilator) [reply src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md:19]: Btw. short and address exist in parallel in some other events, see hackathons

**Review by Freymaurer — APPROVED** [review]

**Comment by Freymaurer** [comment]
pls rebase this PR

**Pull request merged**